src/main/java/com/redhat/cloud/custompolicies/app/rest/PolicyCrudService.java

```diff
@@ -223,10 +223,15 @@ public Response getPoliciesForCustomer() {
              description = "If passed and set to true, the passed policy is also persisted (if it is valid)")
   @APIResponses({
       @APIResponse(responseCode = "500", description = "No policy provided or internal error"),
-      @APIResponse(responseCode = "400", description = "Policy validation failed"),
-      @APIResponse(responseCode = "409", description = "Persisting failed"),
+      @APIResponse(responseCode = "400", description = "Policy validation failed",
+                   content = @Content(schema =@Schema(implementation = String.class,
+                                                                         description = "Reason for failure"))),
+      @APIResponse(responseCode = "409", description = "Persisting failed",
+                   content = @Content(schema =@Schema(implementation = String.class,
```

Comment:
Shouldn't this be `implementation = Msg.class` ?
```suggestion
                   content = @Content(schema =@Schema(implementation = Msg.class,
```